### PR TITLE
add description about Bucket naming in Catalog

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1733,26 +1733,34 @@ Listing the Catalog for existing Buckets is done using the following HTTP reques
...
 Then, we can fetch a particular Workflow based on its `name`:
 
-    GET http://localhost:8080/catalog/buckets/{bucket_id}/resources/{name}
+    GET http://localhost:8080/catalog/buckets/{bucketName}/resources/{name}
 
 NOTE: By default, the Catalog will always return the latest version of the Workflows.
 
+=== Bucket naming requirements
+
+Every bucket is identified by unique bucket name. The bucket naming should follow the following rules:
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: by unique a bucket name

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1733,26 +1733,34 @@ Listing the Catalog for existing Buckets is done using the following HTTP reques
...
+
+* The bucket name can be between 3 and 63 characters long, and can contain only lower-case characters, numbers, and dashes.
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: The bucket name must be between 3 and 63 characters long and can contain only lower-case characters, numbers, and dashes.

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (amouhoub) [reply src/docs/user/ProActiveUserGuide.adoc]: by a unique bucket name

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (amouhoub) [reply src/docs/user/ProActiveUserGuide.adoc]: numbers and dashes (no comma before the last and)

**Pull request merged**